Created new buttonAs example, removed unused scss file and removed old experiments code

micahgodbolt opened the pull request:
Fixed buttonAs interface to work with buttons. Added example using buttonAs, and removing old unused files. 

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

cliffkoh commented on packages/office-ui-fabric-react/src/components/CommandBar/examples/CommandBar.ButtonAs.Example.tsx:
```diff
@@ -0,0 +1,40 @@
+
+    const customButton = (props: IButtonProps) =>
+      <CommandBarButton
+        onMouseEnter={ () => console.log(`${props.text} hovered`) }
```
Even though this is but just an example, we should probably still adhere to the documented best practices and avoid inline functions.

MLoughry commented on packages/office-ui-fabric-react/src/components/CommandBar/examples/CommandBar.ButtonAs.Example.tsx:
```diff
@@ -0,0 +1,44 @@
+import { CommandBar, ICommandBarProps } from 'office-ui-fabric-react/lib/CommandBar';
+import { CommandBarButton, IButtonProps } from 'office-ui-fabric-react/lib/Button';
+
+
```
Build is failing due to multiple consecutive blank lines

micahgodbolt commented:
Fixed.


Sent from Outlook<http://aka.ms/weboutlook>

________________________________
From: Michael Loughry <notifications@github.com>
Sent: Wednesday, May 9, 2018 10:08 AM
To: OfficeDev/office-ui-fabric-react
Cc: Micah Godbolt; Author
Subject: Re: [OfficeDev/office-ui-fabric-react] Created new buttonAs example, removed unused scss file and removed old experiments code (#4820)


@MLoughry commented on this pull request.

________________________________

In packages/office-ui-fabric-react/src/components/CommandBar/examples/CommandBar.ButtonAs.Example.tsx<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FOfficeDev%2Foffice-ui-fabric-react%2Fpull%2F4820%23discussion_r187109685&data=02%7C01%7Cmgodbolt%40microsoft.com%7C85ebd7c21a2b45916ef308d5b5cf8524%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636614825255895073&sdata=R0f4kZxThIJDsQCXFsAD6obHPvtdZZzED2dPVi7IML0%3D&reserved=0>:

> @@ -0,0 +1,44 @@
+import * as React from 'react';
+
+import { CommandBar, ICommandBarProps } from 'office-ui-fabric-react/lib/CommandBar';
+import { CommandBarButton, IButtonProps } from 'office-ui-fabric-react/lib/Button';
+
+


Build is failing due to multiple consecutive blank lines

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FOfficeDev%2Foffice-ui-fabric-react%2Fpull%2F4820%23pullrequestreview-118824102&data=02%7C01%7Cmgodbolt%40microsoft.com%7C85ebd7c21a2b45916ef308d5b5cf8524%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636614825255905078&sdata=oVen%2FdJXd9LjLGARrJu3FysxMUe6P4L%2Fh73nBbTlNmg%3D&reserved=0>, or mute the thread<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FABXlTMDrHREDS-nrF9wPEOKxWf4ZH5ljks5twyKbgaJpZM4T3tpi&data=02%7C01%7Cmgodbolt%40microsoft.com%7C85ebd7c21a2b45916ef308d5b5cf8524%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636614825255905078&sdata=HD1qG01%2BIeKHa55dbUeT21%2FQ%2FfqXN32yX8u2DHkXqN4%3D&reserved=0>.

micahgodbolt commented:
@cliffkoh yeah, caught that in the last lint run. All cleaned up and passing.
